src/server/protocol.ts

```diff
@@ -3206,6 +3206,7 @@ namespace ts.server.protocol {
         placeOpenBraceOnNewLineForFunctions?: boolean;
         placeOpenBraceOnNewLineForControlBlocks?: boolean;
         insertSpaceBeforeTypeAnnotation?: boolean;
+        insertSpaceAftertypeAnnotation?: boolean;

```

Comment:
It seems option has a typo `insertSpaceAftertypeAnnotation`.

Reply:
Oops, my bad!

Reply:
The tests seem to be failing saying 
`Error: The baseline file api/typescript.d.ts has changed.` and
`Error: The baseline file api/tsserverlibrary.d.ts has changed.`,
so I guess I'm not supposed to change these types in the first place?